### PR TITLE
Union of concerned scientists

### PR DESCRIPTION
**Pull request by reasn**
*Please edit everything below that's UPPERCASE and starts with a "📝"*:

**What change is this pull request introducing?**
Adds basic information

**Timestamp**
*Please provide the date on which you discovered the initiative.*
📝YYYY-MM-DD

**References**
*Please add links to references to make sure all content is reliable*
https://en.wikipedia.org/wiki/Union_of_Concerned_Scientists
https://www.ucsusa.org/about-us
https://www.ucsusa.org/sites/default/files/attach/2018/11/annual-report-2018.pdf

### CODE REVIEW

**Review by yeah800 — COMMENTED** [review]
link to resource needed

File: data.js [hunk 1]
```diff
@@ -204,8 +201,37 @@ exports.data = [
...
+            updatedBy: "Alexander", updatedAt: "2019-08-18"
+        },
+        actions: {
+            content: "- Research & Studies" +
+                "\n- PR, lobbying" +
+                "\n- Supporting other NGOs (1,500+)",
+            updatedBy: "Alexander", updatedAt: "2019-08-18",
+        },
+        yearlyIncome: {
+            content: "$ 40M" +
```
Comment (yeah800) [inline data.js]: link needed

File: data.js [hunk 2]
```diff
@@ -204,8 +201,37 @@ exports.data = [
...
+                "\n\n72% membership & contributions" +
+                "\n\n22% foundations",
+            updatedBy: "Alexander", updatedAt: "2018-11-30"
+        },
+        employees: {
+            content: "almost 250 scientists, analysts, policy and communication experts",
```
Comment (yeah800) [inline data.js]: links needed

**Review by yeah800 — COMMENTED** [review]
l mostlty gtm. Please add links

File: data.js (hunk 1 above)
Comment (reasn) [reply data.js]: done

File: data.js (hunk 2 above)
Comment (reasn) [reply data.js]: done

**Review by yeah800 — APPROVED** [review]

**Pull request merged**